external_agent/examples/langgraph_python/README.md

```diff
@@ -0,0 +1,77 @@
+- **Logging and Debugging**: Logging is set up to facilitate debugging and monitoring of the application.
+
+## Security Limitations
+
+Please be aware that this example accepts any API Key or Bearer token for authentication. 
+It is recommended to implement your own authentication security measures to ensure proper security.
+
+## Deployment Instructions
+
+### Step 1: Create a Code Engine Project
+
+1. **Using IBM Cloud Web UI:**
+   - Navigate to [IBM Cloud Code Engine Projects](https://cloud.ibm.com/containers/serverless/projects) and select **Create**. Name your project, for instance `wxo-agent-test1`.
+   - Select the agent you created (`wxo-agent-test1`) and choose the **Application** menu item from the left navigation panel.
+
+2. **Create an API Key for Registry Secret:**
+   - Select **Manage** from the title bar menu and go to **Access (IAM)**.
+   - From the left navigation menu, select **API keys**.
+   - Click **Create** and copy the new API key for use in the registry secret.
+
+3. **Create the Code Engine Application:**
+   - Click the **Create** button to start creating an application.
+   - Under **Code**, select **Build container image from source code**.
+   - In the **Code repo URL** field, enter `https://github.com/watsonx-orchestrate-developer-toolkit`.
+   - Click **Specify build details**:
+     - **SSH secret:** None
+     - **Branch name:** main
+     - **Context directory:** `external_agent/examples/langgraph_python`
+     - Click **Next**
+     - **Dockerfile:** Dockerfile (leave default)
+     - Click **Next**
+     - Under **Registry secret**, create a secret (if one doesn't exist) using the **API Key** created above
+   - **Application name:** Any name, for instance `wxo-agent-test1-app1`
+   - **Domain mappings:** Public
+
+4. **Set Environment Variables:**
+   - Add the following environment variables:
+     - `WATSONX_SPACE_ID`
+     - `WATSONX_API_KEY`
+
+5. **Test the Application:**
+   - Choose **Test application** and click **Application URL**.
+   - Append `/docs` to the end of the URL path to view a formatted API page.
+     - Example: `https://wxo-agent-test1-app1.1pj4w3r1pi47.us-south.codeengine.appdomain.cloud/docs`
+
+### Step 2: Register the New Endpoint as an External Agent
+
+1. **In IBM Watsonx Orchestrate Web UI:**
+   - From the top left hamburger menu, select **Agent Configuration**.
+   - Select **Assistants** from the left-hand navigation.
+   - Click the **Add assistant** button on the top right.
+   - Choose **External Assistant** at the top of the dialog.
+   - Check the **External-agent Assistant** box.
+
+2. **Enter Details:**
+   - **Display Name:** e.g., News Today
+   - **Description:** Enter a description of capabilities.
+   - **API Key:** Enter your API key.
+   - **Service Instance URL:** Use the Test URL with `/chat/completions` appended.
+     - Example: `https://wxo-agent-test1-app1.1pj4w3r1pi47.us-south.codeengine.appdomain.cloud/chat/completions`
```

Comment:
can we have a #3 , calling the agent from orchestrate and include a example input and expected output?